macro/run_simScript.py

```diff
@@ -230,7 +232,8 @@
      DecayVolumeMedium=options.decayVolMed,
      SND=options.SND,
 )
-
+ship_geo.decouple = options.decouple
+ship_geo.psd=2586.0
```

Comment:
This needs a better name and comments to explain what these are, especially if they are set at the top level!